lib/config/fileconf.go

```diff
@@ -258,8 +260,13 @@ func makeSampleSSHConfig(conf *service.Config, enabled bool) SSH {
 				Period:  time.Minute,
 			},
 		}
-		s.Labels = map[string]string{
-			"env": "example",
+		if len(flags.NodeLabels) > 0 {
+			labels := strings.Split(flags.NodeLabels, ",")
+			s.Labels = make(map[string]string)
+			for _, label := range labels {
+				parts := strings.Split(label, ":")
+				s.Labels[parts[0]] = parts[1]
```

Comment:
I think `parts[1]` is going to panic with index out of range if `label` doesn't actually contain `:`.

What do you think about failing with an error instead? Perhaps something in the style of "invalid node label \<label\>; labels must follow the format of key:value".

Reply:
Good call, I'll expand this function to support returning error messages

Reply:
done